doc/introduction/interfaces/numpy.rst

```diff
@@ -110,14 +110,66 @@ with respect to both QNode parameters ``phi`` and ``theta``:
 
     phi = np.array([0.5, 0.1])
     theta = 0.2
-    dcircuit = qml.grad(circuit3, argnum=[0, 1])
+    dcircuit = qml.grad(circuit3)
 
 Evaluating this gradient function at specific parameter values:
 
 >>> dcircuit(phi, theta)
 (array([ -4.79425539e-01,   1.11022302e-16]), array(0.0))
 
 
+Non-differentiable arguments
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Note that **all positional arguments provided to the QNode are assumed to be differentiable
+by default**:
+
+>>> from pennylane import numpy as np
+>>> np.array([0.1, 0.2])
+tensor([0.1, 0.2], requires_grad=True)
```

Comment:
```suggestion

```
However, you may want to pass arguments as positional arguments to a QNode but _not_ have PennyLane
consider them when computing gradients. To accomplish this, it is important to know that importing numpy from PennyLane has an important side effect: creating of arrays with ``np.array()`` produces 
a _tensor_ which behaves like "normal" numpy arrays, but has a special flag indicating whether it is trainable or not. 

>>> from pennylane import numpy as np
>>> np.array([0.1, 0.2])
tensor([0.1, 0.2], requires_grad=True)
```